recipe/meta.yaml

```
@@ -1,45 +1,133 @@
 {% set name = "testtools" %}
-{% set version = "2.4.0" %}
-{% set bundle = "tar.gz" %}
-{% set hash_type = "sha256" %}
-{% set hash = "64c974a6cca4385d05f4bbfa2deca1c39ce88ede31c3448bee86a7259a9a61c8" %}
-{% set build = 0 %}
+{% set version = "2.7.2" %}
 
 package:
-  name: {{ name }}
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.{{ bundle }}
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ bundle }}
-  {{ hash_type }}: {{ hash }}
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5be5bbc1f0fa0f8b60aca6ceec07845d41d0c475cf445bfadb4d2c45ec397ea3
 
 build:
-  number: {{ build }}
-  script: python setup.py install --single-version-externally-managed --record=record.txt
+  number: 0
+  skip: true  # [py<38]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
 
 requirements:
   host:
+    - pip
     - python
-    - setuptools
-    - pbr >=0.11
-
+    - setuptools >=61
+    - hatchling
+    - hatch-vcs
+    - wheel
```

```suggestion
    - hatchling
    - hatch-vcs
```
If `hatchling` is a build backend, then `setuptools` and `wheel` aren't needed. `hatchling` has its own WheelBuilder.
Technically, setuptools is redundant even if the maintainers mention it.

---

It appears that the maintainers have finally removed it https://github.com/testing-cabal/testtools/commit/8dc76f83808eb6c4c5a4216b226f0ca230997f9c.